tests/test_hgvs_dataproviders_uta.py

```diff
@@ -66,7 +66,19 @@ def test_get_tx_exons_invalid_alt_aln_method(self):
 
     def test_get_tx_for_gene(self):
         tig = self.hdp.get_tx_for_gene("VHL")
-        self.assertEqual(16, len(tig))
+        
+        expected_data = [
+            ("NM_001354723.1", "NC_000003.11"),
+            ("NM_198156.2", "NC_018914.2"),
+            ("ENST00000345392", "NC_000003.11"),
+        ]
+        found = 0
+        for hgnc, cds_start_i, cds_end_i, tx_ac, alt_ac, alt_aln_method in tig:
```

Comment:
hgnc, cds_start_i, cds_end_i, alt_aln_method aren’t used 
```suggestion
        for _, _, _, tx_ac, alt_ac, _ in tig:
```